package.json

```diff
@@ -46,7 +46,7 @@
   "dependencies": {
     "@commander-js/extra-typings": "^11.0.0",
     "@cowprotocol/contracts": "^1.4.0",
-    "@cowprotocol/cow-sdk": "^5.2.0",
+    "@cowprotocol/cow-sdk": "^5.3.0-RC.2",
```

Comment:
Is the intent to keep this as a release candidate, or is the intent to bump this to 5.3.0 when it's final?

Reply:
Yes, lets do it

Reply:
Okay, @alfetopito  I see you went already for it https://www.npmjs.com/package/@cowprotocol/cow-sdk?activeTab=versions

Reply:
Yes, sorry forgot to comment in this thread